backend/api/views.py

```diff
@@ -51,6 +51,14 @@ def put(self, request, *args, **kwargs):
 class EventListView(generics.ListAPIView):
     queryset = Event.objects.all()
     serializer_class = EventSummarySerializer
+    filter_backends = (filters.SearchFilter,)
+    search_fields = ('title','description')
+    method_serializer_classes = {
```

Comment:
Same comment, you don't need neither `method_serializer_classes` nor `def search`.